src/main/java/seedu/nuscents/parser/Parser.java

```diff
@@ -78,6 +80,8 @@ public static <TransactionList> Command parseCommand(String text, TransactionLis
                 return new ViewCommand(parseTaskIndex(arguments));
             case COMMAND_FILTER:
                 return new FilterCommand(parseFilterCategory(arguments));
+            case COMMAND_EDIT:
+                return parseEdit(arguments);
```

Comment:
Perhaps you could follow the format of the other commands, that is, return new EditCommand(parseEdit(arguments))